Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -312,6 +312,33 @@ public class LottieAnimationLayer: CALayer {
     }
   }
 
+  public override var anchorPoint: CGPoint {
+    get {
+      if let animationLayer = animationLayer {
```

Comment:
You'd want to create a new internal `var rootAnimationLayer: RootAnimationLayer?` stored property (since `RootAnimationLayer` is internal) and then change `animationLayer` to `public var animationLayer: CALayer? { rootAnimationLayer }`

Reply:
You'll also want to add a comment mentioning when `animationLayer` is created and what triggers it to be replaced with a separate `CALayer` instance.

Reply:
@calda Oh yeah. I like that approach. Makes sense and that way we don't need to keep exposing CALayer properties when somebody needs it.

I added the changes in the new commit, and removed the old changes from the first commit. Let me know if that works for you.